src/main/java/com/synopsys/integration/alert/common/descriptor/config/context/DescriptorActionApi.java

```diff
@@ -41,13 +46,44 @@
 
 public abstract class DescriptorActionApi {
 
-    public abstract void validateConfig(final FieldModel fieldModel, final Map<String, String> fieldErrors);
+    public static final String REQUIRED_FIELD_MISSING = "Required field missing";
+
+    public void validateConfig(final Collection<ConfigField> descriptorFields, final FieldModel fieldModel, final Map<String, String> fieldErrors) {
+        for (final ConfigField field : descriptorFields) {
```

Comment:
Since we know what field this is, I think we can get a little more explicit about which required field is missing. Then we don't have to do extra validation operations in the various UIConfig classes with `.StringUtils.isBlank(...)`. Or am I missing something?